src/test/scala/parsec/TestCharParsers.scala

```diff
@@ -39,10 +39,9 @@ class CharParsersSuite
   test("can parse a word, a digit, and another letter") {
     val (strZ, strCombine) = stringFolder
 
-    val wordDigitLetter: Parser[(String, (Char, Char))]
-      = (letters ~ digit ~ letter).fold(strZ, strCombine) map {
-          case (ls, other) => (ls.toString, other)
-        }
```

Comment:
apart from those two, LGTM. Please make the required changes, I'll merge the PR :-)